### PR TITLE
[service.librarywatchdog] 1.0.0

### PR DESCRIPTION
**Pull request by phunkyfish**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by phunkyfish** [comment]
3 of 3

File: service.librarywatchdog/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
+v1.0.0
```
Comment (enen92) [inline service.librarywatchdog/changelog.txt]: please use the `<news>` element in addon xml only for the last(ish) version of the changelog and remove the changelog.txt from the submission if possible. As the changelog is no longer used on the interface, there's no reason for us to ship it from the infrastruture.

Comment (phunkyfish) [reply service.librarywatchdog/changelog.txt]: Done ;)

**Comment by enen92** [comment]
@phunkyfish all sorted at least :)

**Comment by phunkyfish** [comment]
Thanks for sorting this out 😃

**Pull request merged**